src/App.jsx

```diff
@@ -11,6 +13,7 @@ const App = () => {
       </header>
       <main className="main">
         <Outlet />
+        <HeaderSearchBar />
```

Comment:
create a page folder under routes and place your component inside that page.
Then @ index.jsx, add a path: "stories/header-search-bar" with element of the page 
Lastly, under story-page.jsx, add a Link to your component